IM_calculation/Advanced_IM/advanced_IM_factory.py

```diff
@@ -18,7 +22,7 @@
 
 
 def get_config(config_file=CONFIG_FILE_NAME):
-    return load_yaml(config_file)
+    return utils.load_yaml(config_file)
```

Comment:
Continuing my crusade against `qcore.utils.load_yaml`. It's redundant, just replace it with `yaml.safe_load`